pandas/util/clipboard.py

```diff
@@ -1,266 +1,108 @@
-# Pyperclip v1.5.15
-# A cross-platform clipboard module for Python.
```

Comment:
i have removed the file pandas\util\clipboard\clipboard.py and replaced it with "__init__.py" that is shipped with pyperclip. So these comments are now in the "__init__.py file". This change is showing that this file has been removed